app/views/account/SubscriptionView.coffee

```diff
@@ -179,20 +179,20 @@ class PersonalSub
     removeSub = =>
       payPalInfo = me.get('payPal')
       stripeInfo = _.clone(me.get('stripe'))
-      if stripeInfo
-        delete stripeInfo.planID
-        me.set('stripe', stripeInfo)
-        me.once 'sync', ->
-          window.tracker?.trackEvent 'Unsubscribe End', message: message
-          document.location.reload()
-        me.patch({headers: {'X-Change-Plan': 'true'}})
-      else if payPalInfo?.billingAgreementID
+      if payPalInfo?.billingAgreementID
         api.users.cancelBillingAgreement({userID: me.id, billingAgreementID: payPalInfo?.billingAgreementID})
         .then (response) =>
           window.tracker?.trackEvent 'Unsubscribe End', message: message
           document.location.reload()
         .catch (jqxhr) =>
           console.error('PayPal unsubscribe', jqxhr)
+      else if stripeInfo
+        delete stripeInfo.planID
+        me.set('stripe', stripeInfo)
+        me.once 'sync', ->
+          window.tracker?.trackEvent 'Unsubscribe End', message: message
+          document.location.reload()
+        me.patch({headers: {'X-Change-Plan': 'true'}})
```

Comment:
Why was this change made?

Reply:
We'll start leaving users with strip and payPal properties.  Probably not a huge deal, but it broke this specific if/else logic.